rs/state_machine_tests/src/lib.rs

```diff
@@ -2442,9 +2446,16 @@ impl StateMachine {
         SystemTime::UNIX_EPOCH + Duration::from_nanos(self.time.load(Ordering::Relaxed))
     }
 
-    /// Returns the state machine time at the beginning of next round.
+    /// Returns the state machine time at the beginning of next round
+    /// under the assumption that time won't be advanced before that
+    /// round is finalized.
     pub fn time_of_next_round(&self) -> SystemTime {
```

Comment:
Side note: The fact that there is a `time_of_next_round` and a different `get_time_of_next_round` just below is a bit weird.

Reply:
Is it confusing that there are two function calls or that there are calls to functions with different names (or both)?

Reply:
Mainly that there are two functions with almost the same name and almost the same functionality, except the return type. But this isn't something that needs changing, I was just confused at first.